uber/tests/models/test_getattr_fields.py

```diff
@@ -10,6 +10,7 @@ def test_ints():
 def test_label():
     assert '' == Attendee(paid=None).paid_label
     assert 'yes' == Attendee(paid=c.HAS_PAID).paid_label
+    assert 'none' == Attendee(paid="none").paid_label
```

Comment:
Yessss, this unit test was for old behavior that would spit out the value given if it wasn't an integer. However, later when doing CSV exports I found that doing this would make importing data impossible later on, so I changed it back to returning an empty string. I forgot to fix the unit test since it was in a different branch.

So it should be `assert '' == Attendee(paid="none").paid_label` now.